static/src/components/TaskView.vue

```diff
@@ -66,15 +72,15 @@ export default defineComponent({
           :label="label"
           :is-attentive="isAttentive"
           :is-trained="isTrained"
-          :competency="competency"
-          :task-id="task.id"
-          :image-id="imageId"
+          :competency="'Prof. Dr. Med.'"
```

Comment:
Any particular reason why that is now again hardcoded inside the component and not coming from outside? Or just temporary, as we have not dealt with the competency seriously yet?